src/sentry/apidocs/docs/organization_release_details_schema.py

```diff
@@ -0,0 +1,173 @@
+ORGANIZATION_RELEASE_DETAILS_SCHEMA = {
```

Comment:
i made this file because there is no serializer for the get response & the actual schema is very large, so defining it using OpenAPI was the easiest.

Reply:
One goal of these changes are that we have docs that change dynamically when code changes happen. I think if someone change the return type of the endpoint the docs won't get updated this way. Can we have a typedict instead which we can use for typing in the code as well as docs?